docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '2'
```

Comment:
Can't believe we didn't have this before. Such a good thing

Reply:
Nice